selfdrive/car/ford/carcontroller.py

```diff
@@ -46,33 +31,33 @@ def update(self, CC, CS):
 
     ### acc buttons ###
     if CC.cruiseControl.cancel:
-      can_sends.append(fordcan.create_button_command(self.packer, CS.buttons_stock_values, cancel=True))
-      can_sends.append(fordcan.create_button_command(self.packer, CS.buttons_stock_values, cancel=True, bus=CANBUS.main))
+      can_sends.append(self.ford_can.create_button_msg(CS.buttons_stock_values, cancel=True))
+      can_sends.append(self.ford_can.create_button_msg(CS.buttons_stock_values, cancel=True, bus=CANBUS.main))
     elif CC.cruiseControl.resume and (self.frame % CarControllerParams.BUTTONS_STEP) == 0:
-      can_sends.append(fordcan.create_button_command(self.packer, CS.buttons_stock_values, resume=True))
-      can_sends.append(fordcan.create_button_command(self.packer, CS.buttons_stock_values, resume=True, bus=CANBUS.main))
+      can_sends.append(self.ford_can.create_button_msg(CS.buttons_stock_values, resume=True))
+      can_sends.append(self.ford_can.create_button_msg(CS.buttons_stock_values, resume=True, bus=CANBUS.main))
     # if stock lane centering isn't off, send a button press to toggle it off
     # the stock system checks for steering pressed, and eventually disengages cruise control
     elif CS.acc_tja_status_stock_values["Tja_D_Stat"] != 0 and (self.frame % CarControllerParams.ACC_UI_STEP) == 0:
-      can_sends.append(fordcan.create_button_command(self.packer, CS.buttons_stock_values, tja_toggle=True))
+      can_sends.append(self.ford_can.create_button_msg(CS.buttons_stock_values, tja_toggle=True))
 
 
     ### lateral control ###
-    if CC.latActive:
-      lca_rq = 1
-      apply_angle = apply_ford_steer_angle_limits(actuators.steeringAngleDeg, self.apply_angle_last, CS.out.vEgo)
-    else:
-      lca_rq = 0
-      apply_angle = 0.
-
     # send steering commands at 20Hz
     if (self.frame % CarControllerParams.LKAS_STEER_STEP) == 0:
-      # use LatCtlPath_An_Actl to actuate steering
-      path_angle = math.radians(apply_angle) / CarControllerParams.LKAS_STEER_RATIO
+      if CC.latActive:
+        # use LatCtlCurv_No_Actl to actuate steering
+        # IMPORTANT! left is positive
```

Comment:
keep comments like this in one place. right now, some are here and others in fordcan.py